packages/functional-tests/lib/targets/remote.ts

```diff
@@ -7,12 +7,11 @@ export abstract class RemoteTarget extends BaseTarget {
     password: string,
     options?: any
   ): Promise<Credentials> {
-    const creds = await this.auth.signUp(email, password);
+    const creds = await this.auth.signUp(email, password, options || {});
     const code = await this.email.waitForEmail(
       email,
       EmailType.verify,
-      EmailHeader.verifyCode,
-      ...(options || {})
+      EmailHeader.verifyCode
```

Comment:
Good catch. I have no clue why I tried spreading the options argument here... I think that was just a straight up mistake. I had a pretty tough time migrating these tests, and this was probably just some hangfire from one many failed attempts.